source/_posts/2015-10-09-the-slack-users-code.markdown

```diff
@@ -0,0 +1,258 @@
+---
+layout: post
+title: "The Slack Users' Code"
+date: 2015-10-09 22:30:00 -0500
+comments: true
+categories:
+---
+Without a code, [Slack](https://slack.com/) would be chaos. These are my ideas
+about what makes a good Slack team and Slack citizen. These are for _work Slack
+teams_. If you have a social Slack team set up, you do you because the
```

Comment:
I might italicize "you do you" so it's clearer that you're referencing an idiom or whatever you call that linguistic construct. As written I had to read it 3 times before I figured out what you were saying.